isotpsniffer.c

```diff
@@ -197,7 +199,7 @@ int main(int argc, char **argv)
 	unsigned char buffer[4096];
```

Comment:
Can you add some
#define PDU_BUF_SIZE 8300
here which is then used for buffer[PDU_BUF_SIZE]  and in the checks with '4095' below.